e2e-tests/wait-for-start.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
nit: license message and a short description of what this file is for.

Reply:
Done